packages/Webkul/BookingProduct/src/Resources/lang/uk/app.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+return [
+    'admin' => [
+        'catalog' => [
+            'products' => [
+                'booking' => 'Інформація про запис',
+                'booking-type' => 'Тип запису',
+                'default' => 'За замовчуванням',
+                'appointment-booking' => 'Запис на прийом',
+                'event-booking' => 'Забронювати подію',
+                'rental-booking' => 'Забронювати оренду',
+                'table-booking' => 'Забронювати столик',
+                'slot-duration' => 'Довжина слота (у хвилинах)',
+                'break-time' => 'Перерва між слотами (у хвилинах)',
+                'available-every-week' => 'Доступно щотижня',
+                'yes' => 'Так',
+                'no' => 'Ні',
+                'available-from' => 'Доступно з',
+                'available-to' => 'Доступно по',
+                'same-slot-all-days' => 'Один слот на всі дні',
+                'slot-has-quantity' => 'У слота вже є запис',
+                'slots' => 'Слоти',
+                'from' => 'Від',
+                'to' => 'До',
+                'qty' => 'Кількість',
+                'add-slot' => 'Додати слот',
+                'sunday' => 'Неділя',
+                'monday' => 'Понеділок',
+                'tuesday' => 'Вівторок',
+                'wednesday' => 'Середа',
+                'thursday' => 'Четвер',
+                'friday' => 'П'ятниця',
```

Comment:
Note that using the single quote within a string enclosed in single quotes requires escaping via backslash like that:

```suggestion
                'friday' => 'П\'ятниця',
```

The same is true for other similar messages.